src/routes/others.js

```diff
@@ -0,0 +1,15 @@
+import express from 'express';
+
+const otherRouter = express.Router();
```

Comment:
Mad props! for taking the initiative to create this router, especially the catch-all handler on line `10`.